### ISSUE TITLE
Pending and Current Operation View

### ISSUE THREAD

**Issue by OssNass** [elementary/appcenter#1368]
Can you add an extra view to the app that shows the progress of current operation and the queue of other pending operations, of all types (install, update or remove)?

It should appear as an extra button right next to installed called "Operations", when clicking on it, it shows the current operation with progress bar and the other pending operation with cancel button for both current and pending operation

**Comment by askanish** [comment]
Yes will definitely need something like this when multiple applications are being installed. Need a place to see progress of all apps
